### ISSUE TITLE
CODE

### ISSUE THREAD

**Issue by wqdong8** [lodurality/anise_paper_code#1]
HI, nice work! Will the code for preprocess part data be released?

**Comment by lodurality** [comment]
Yes, this code is long overdue and I am planning to release it in next 2 weeks or so.

**Comment by wqdong8** [comment]
Thanks for your response, looking forward to your code~
